src/main/java/seedu/address/logic/commands/ListClassCommand.java

```diff
@@ -12,13 +14,34 @@
 
     public static final String COMMAND_WORD = "list_class";
 
-    public static final String MESSAGE_SUCCESS = "Listed all classes";
+    public static final String MESSAGE_SUCCESS_ALL = "Listed all classes";
+
+    public static final String MESSAGE_SUCCESS = "Listed all classes on %1$s";
```

Comment:
```suggestion
    public static final String MESSAGE_SUCCESS_DAY = "Listed all classes on %1$s";
```

Reply:
May be clearer this way about the purpose of having 2 different messages

Reply:
Yup much clearer. Thanks!